src/search.ts

```diff
@@ -99,6 +105,16 @@ export async function findFilesToUpload(
     if (!fileStats.isDirectory()) {
       debug(`File:${searchResult} was found using the provided searchPath`)
       searchResults.push(searchResult)
+
+      // detect any files that would be overwritten because of case insensitivity
+      const currentSetSize = set.size
+      set.add(searchResult.toLowerCase())
```

Comment:
I am not sure if this will work across all encodings, but since we are just outputting a line, I guess it's okay.  

Also, since what you are really interested in is if this `set` contains the file, why not just call `has` on it to be explicit?  I think the intention is clearer with `has` or `contains` checks.

Reply:
Changed to use `has`